src/enums/nodeLocationType.ts

```diff
@@ -0,0 +1,7 @@
+enum NodeLocationType {
```

Comment:
There already was a type for this exact purpose. Use that or replace it if necessary.

Reply:
fixed